### PR TITLE
swift 5 fixes

### PR DESCRIPTION
**Pull request by iam-bartl**

### CODE REVIEW

**Review by MarcSteven — APPROVED** [review]
Yeah, cool improvement

File: RHSideButtons/RHSideButtons/RHSideButtons.swift [hunk 1]
```diff
@@ -211,8 +212,6 @@ extension RHSideButtons: RHButtonViewDelegate {
             state = .hidden
         } else {
             state = state == .shown ? .hidden : .shown
-            
-            delegate?.sideButtons(self, didTriggerButtonChangeStateTo: state)
```
Comment (robertherdzik) [inline RHSideButtons/RHSideButtons/RHSideButtons.swift:215]: Why we are removing in here delegate call? 🤔

**Review by robertherdzik — COMMENTED** [review]
Great initiative! Thanks for participating into my project 🙌 What you think, to use Xcode option to convert to latest swift version: Edit->Convert->To Current Swift Syntax..

In this way the procedure is less error prone and automatically pbxproj will be updated with Swift 5

File: RHSideButtons/RHSideButtons/RHSideButtons.swift (hunk 1 above)
Comment (iam-bartl) [reply RHSideButtons/RHSideButtons/RHSideButtons.swift:215]: emm :) ... it's modification for my own purposes. i didn't want to add this to the pull request :)
Btw, I dropped support of RHSideButtons in my project, so you can decline my pull request

Comment (iam-bartl) [reply RHSideButtons/RHSideButtons/RHSideButtons.swift:215]: Or I could reopen other PR from different branch

Comment (robertherdzik) [reply RHSideButtons/RHSideButtons/RHSideButtons.swift:215]: Ok, I will decline PR, Thanks ;)

**Pull request closed without merging**